docs/app-integrations/gmail.mdx

```diff
@@ -32,19 +32,40 @@ parameters = {
 <Tip>
 You need a Google account in order to use this integration. Here is how to get the credentials file:
 
-1. Go to https://console.cloud.google.com/apis/dashboard.
-2. In the `Enabled APIs & services` tab, click on `Enable APIs and Services` button, search for `Gmail API`, and enable it.
-3. In the `OAuth consent screen` tab, create a new project and an app:
-    - name your project,
-    - choose user type as external,
-    - add app name and user support email,
-    - add the following scopes: `.../auth/userinfo.profile`, `.../auth/userinfo.email`, `openid`,
-    - add a test user,
-    - save and go back to the dashboard.
-4. In the `Credentials` tab, click on `Create Credentials` button and choose `OAuth client ID`:
-    - choose application type as `Desktop app` and give it a name,
-    - click on `Create`,
-    - download the JSON file.
+1. Create a Google Cloud Platform (GCP) Project:
+
+    1.1 Go to the GCP Console (https://console.cloud.google.com/).
+    
+    1.2 If you haven't created a project before, you'll be prompted to do so now.
+    
+    1.3 Give your new project a name.
+    
+    1.4 Click `Create` to create the new project.
+
+2. Enable the Gmail API:
+
+    2.1 In the GCP Console, select your project.
+
+    2.2 Navigate to `APIs & Services` > `Library`.
```

Comment:
Actually, that's the tab name:
2.2 Navigate to `APIs & Services` > `Enabled APIs & services`.